scripts/remove

```diff
@@ -3,13 +3,26 @@
 source _common.sh
 source /usr/share/yunohost/helpers
 
-ynh_script_progression --message="Removing service in YunoHost..." --weight=1
-yunohost service remove $app
+#=================================================
+# REMOVE SYSTEM CONFIGURATIONS
+#=================================================
+# REMOVE SYSTEMD SERVICE 
+#=================================================
+ynh_script_progression --message="Removing system configurations related to $app..." --weight=1
+
+# Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
+if ynh_exec_warn_less yunohost service status $app >/dev/null
+then
+	ynh_script_progression --message="Removing $app service integration..." --weight=1
+	yunohost service remove $app
+fi
```

Comment:
```suggestion
# Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
if ynh_exec_warn_less yunohost service status $app >/dev/null
then
	ynh_script_progression --message="Removing $app service integration..." --weight=1
	yunohost service remove $app
fi

ynh_script_progression --message="Removing system configurations related to $app..." --weight=1
```

Reply:
Thanks for you contribution. If I'm not wrong you just moved the line `ynh_script_progression --message="Removing system configurations related to $app..." --weight=1` after the `if` block!?

Reply:
Yeah, indeed :) It’s just to have more consistent progression messages: the "removing app service integration" message will be visible while the other system configurations will be actually removed. It’s a nitpick :D